guides/source/configuring.md

```diff
@@ -1024,6 +1023,9 @@ development:
 
 If your development database has a root user with an empty password, this configuration should work for you. Otherwise, change the username and password in the `development` section as appropriate.
 
+NOTE: Rails 6 drops supporting MySQL 5.1, supports MySQL 5.5.8 or higher.
```

Comment:
I think "Rails 6 drops supporting MySQL 5.1" part is unnecessary for the guide, since we also didn't mention about which Rails version supports MySQL 5.0.

Reply:
Make sense. Updated and made some additional changes.